packages/demo/package.json

```diff
@@ -7,15 +7,19 @@
   },
   "babel": {
     "presets": [
-      "env"
+      "@deloitte-digital-au/babel-preset"
     ]
   },
   "browserslist": [
     "last 2 versions",
     "IE 11"
   ],
   "devDependencies": {
-    "@deloitte-digital-au/webpack-config": "^0.0.5"
+    "@deloitte-digital-au/webpack-config": "^0.0.5",
+    "@deloitte-digital-au/babel-preset": "^0.0.5"
```

Comment:
This is a mistake, I will fix but dont need to include babel-preset here as it comes with webpack-config.